charts/consul/templates/gateway-resources-configmap.yaml

```diff
@@ -121,5 +121,8 @@ data:
             {{- end }}
         spec:
           gatewayClassName: consul-mesh-gateway
+          listeners: 
+          - name: "wan"
```

Comment:
Should this value be hardcoded here or in the service.go file?

Reply:
https://hashicorp.atlassian.net/browse/NET-6822 From the ticket, I believe we want it to be defined on the CRD